src/Pages/GlobalConfigurations/ClustersAndEnvironments/ClusterEnvironmentDrawer/ClusterEnvironmentDrawer.tsx

```diff
@@ -226,21 +226,32 @@ export const ClusterEnvironmentDrawer = ({
     return (
         <Drawer position="right" width="800px" onEscape={hideClusterDrawer} onClose={hideClusterDrawer}>
             <div className="h-100 bcn-0 flexbox-col" onClick={stopPropagation}>
-                <div className="flexbox dc__align-items-center dc__content-space dc__border-bottom bcn-0 py-12 px-20">
-                    <h3 className="m-0 fs-16 fw-6 lh-1-43">{id ? 'Edit Environment' : 'Add Environment'}</h3>
-                    <button
-                        type="button"
-                        aria-label="close-btn"
-                        className="dc__transparent flex icon-dim-24"
-                        onClick={hideClusterDrawer}
-                    >
-                        <Close className="icon-dim-24 dc__align-right cursor" />
-                    </button>
-                </div>
                 <form
-                    className="flex-grow-1 flexbox-col"
+                    className="flex-grow-1 flexbox-col dc__overflow-scroll"
                     onSubmit={handleSubmit(namespaceLabels.labels ? withLabelEditValidation : onValidation())}
                 >
+                    <div className="flexbox dc__align-items-center dc__content-space dc__border-bottom bcn-0 py-12 px-20">
```

Comment:
Why are we moving it inside the form?

Reply:
As with h-100 div was overflowing because to flex column caveat